listenbrainz/config.py.sample

```diff
@@ -157,3 +157,6 @@ SESSION_REMEMBER_ME_DURATION = 365
 
 # listen dumps creation dir
 LISTEN_DUMP_TEMP_DIR_ROOT = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'tmp')
+
+REJECT_LISTENS_WITHOUT_USER_EMAIL = False
+REJECT_USERS_WITHOUT_EMAIL = False
```

Comment:
These items should have documentation explaining what they do.

Reply:
Setting this item to True will prevent a user from logging in. I wonder if we should have an intermediate step here where from release date we always show a warning to users who have no email, but don't prevent them from logging in until the date that we set `REJECT_LISTENS_WITHOUT_USER_EMAIL` to True

Reply:
Added docs on the options work. As discussed elsewhere, existing users without email will always be able to login.